qiskit/dagcircuit/dagdependency.py

```diff
@@ -382,8 +383,12 @@ def _create_op_node(self, operation, qargs, cargs):
         Returns:
             DAGDepNode: the newly added node.
         """
-        directives = ["measure"]
-        if not getattr(operation, "_directive", False) and operation.name not in directives:
+        directives = []
+        if (
+            not getattr(operation, "_directive", False)
+            and not isinstance(operation, Measure)
+            and operation.name not in directives
+        ):
```

Comment:
After introducing the `isinstance` check to harness subclassing, directives became empty. Should it be removed? If kept, I would suggest turning it into a `set` instead of a `list`.